README.md

```diff
@@ -150,6 +145,8 @@ Currently, the following high-level languages has been implemented for LLB:
 -   [Mockerfile](https://matt-rickard.com/building-a-new-dockerfile-frontend/)
 -   [Gockerfile](https://github.com/po3rin/gockerfile)
 -   [bldr (Pkgfile)](https://github.com/talos-systems/bldr/)
+-   [HLB](https://github.com/openllb/hlb)
```

Comment:
Neither HLB or Earthly expose a reusable frontend atm afaik. Maybe "Used by" section instead because of that? If you don't think that distinction matters then I don't care much actually

Reply:
the document just say `implemented for LLB`, not saying frontend is exposed

Reply:
ok

Reply:
We do want to move HLB to a frontend but we'll need `Export` in the gateway API, otherwise we have full compatibility I think. Hopeful because we've had some conversations around that.